resources/todomvc/big-dom-generator/javascript-web-components/generated-variables.css

```diff
@@ -0,0 +1,50 @@
+:root {
```

Comment:
I'm not sure if we'll get away with having more generated files outside the dist folder.

Reply:
It is the same format that we used for angular, we will move if needed